import java.util.Scanner;

public class T14681 {
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        int x = sc.nextInt();
        int y = sc.nextInt();

        if(x != 0){
            if (y != 0){
                // 1사분면 일때
                if (x > 0 && y > 0){
                    System.out.println(1);
                } else if(x < 0 && y > 0){
                    System.out.println(2);
                } else if(x < 0 && y < 0){
                    System.out.println(3);
                } else {
                    System.out.println(4);
                }

            } else {
                System.out.println("이것은 불가능합니다.");
            }
            
        } else {
            System.out.println("이것은 불가능합니다.");
        }
    }
}
